graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -162,6 +162,8 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
    * in the graph.
    * @param negative_ratio The ratio of the number of negative samples to
    * positive samples.
+   * @param negative_ratio The total count of nodes that can be selected.
```

Comment:
```suggestion
   * @param num_nodes The total count of nodes that can be selected.
```

Reply:
total count of dst nodes per src node?

Reply:
We need better name than num_nodes.

Reply:
In general it use `negative_ratio` in GNN. And in the future we may support `float` type, so a ratio is more fit?

Reply:
This does have some ambiguity. Actually it means the max node id can be selected from, ususally it is the number of a specific type node. So use `max_node_id  ` is better?